### DISCUSSION TITLE
Should be good having a better editor for HTML inserted in Resend, due problem of inline editing

### DISCUSSION THREAD

**Discussion by julioflima** [resend/react-email#1670] [Ideas]
### Goals

1. Let the the user use just Resend to do everything related with the creating and send emails to audience;
3. User can block from editing the HTML tags and let it just edit the TEXT inside of HTML.
4. User can have a built in code editor, with linting to say what is breaking in the HTML.
5. User can have a wider editor with the best theme in world, `Dracula`. Or another.

### Background

In now days is not useful the the Editor of resend.

HTML is too verbose, and HTML for email even more, because the style is inline, with a short editor with no linting, is easy to commit a mistake and break the HTML.

### Proposal

Deliver in the screen in below an awesome built in code editor for HTML, with all features and goals quoted above:

<img width="1470" alt="image" src="https://github.com/user-attachments/assets/308187ea-a69b-4a7c-bbd3-4c1a4df1d774">

**Comment by gabrielmfern** [comment]
Nice idea, but the Resend folks won't see this one from this discussion, try sending this as an email to support@resend.com
